share/pot/set-status.sh

```diff
@@ -43,9 +43,13 @@ _set_status()
 
 	echo "pot.status=$_new_status" >> "$_status_file"
 	# remove first (and outdated) occurrence of pot.status
-	${SED} -i '' -n -e ":a" \
-		-e '/^pot\.status=/{n;bc' -e ':c' -e 'p;n;bc' -e '}' \
-		-e "p;n;ba" "$_status_file"
+	if [ "$(grep -c "^pot\.status=" "$_status_file")" -gt 1 ]; then
+		${SED} -i '' -e '1d' "$_status_file"
```

Comment:
```suggestion
	${SED} -i '' -n -e ":a" \
		-e '/^pot\.status=/{n;bc' -e ':c' -e 'p;n;bc' -e '}' \
		-e "p;n;ba" "$_status_file"'
```

This (combined with your new check) should work fine. It always removes the first status line (but it doesn't figure out that there is only one, which is what your grep fixes now). >_<